projects/storefrontlib/src/cms-components/checkout/components/review-submit/review-submit.component.ts

```diff
@@ -237,4 +264,36 @@ export class ReviewSubmitComponent {
     const step = this.checkoutStepService.getCheckoutStep(stepType);
     return step && step.routeName;
   }
+
+  shippingSteps(steps: CheckoutStep[]): CheckoutStep[] {
+    return steps.filter((step) => this.isShippingStep(step));
+  }
+
+  paymentSteps(steps: CheckoutStep[]): CheckoutStep[] {
+    return steps.filter((step) => this.isPaymentStep(step));
+  }
+
+  protected isShippingStep(step: CheckoutStep): boolean {
```

Comment:
LGTM, but what do you think about the following:

without create additional functions for this, what do you think about `shippingSteps.includes(step.type[0])` in the filter, where shippingSteps is a `const of array`  with values of those enum types

Reply:
in the end, it's the same but we won't have a need to create additional helper functions